portfolio/src/main/webapp/script.js

```diff
@@ -46,6 +47,16 @@ function createListElement(text) {
   return liElement;
 }
 
+/** Checks if the user has inputted any text upon clicking submit. */
+function isInputEmpty() {
+  const comment = document.getElementById('text-input').value;
+  if (comment == null || comment === '') {
```

Comment:
if (!comment || comment === '')
therefore you check whether comment is null or undefined.

And do you want to check whether the comment contains only spaces?